src/libsyntax_pos/symbol.rs

```diff
@@ -404,9 +404,11 @@ declare_keywords! {
     (49, Virtual,            "virtual")
     (50, Yield,              "yield")
 
+    // Edition-specific keywords used in the language.
+    (51, Dyn,                "dyn") // >= 2018 Edition only
+
     // Edition-specific keywords reserved for future use.
```

Comment:
Comment is a bit misleading; these keywords are used on nightly... clarify that maybe?